test/tools/WebListener/Controllers/CertController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using mvc.Models;
+using System.Collections;
+
+namespace mvc.Controllers
+{
+    public class CertController : Controller
+    {
+        public ActionResult Index()
+        {
+            // X509Certificate2 objects do not serialize as JSON. Create a HashTable instead
+            Hashtable output = new Hashtable 
+            {
+                {"Status", "FAILED"}
+            };
```

Comment:
Please remove ';'. Below too.

Reply:
I still see ';'.

Reply:
The Semicolon has to be there. When I tried to remove it I get compile errors saying it is missing.